### ISSUE TITLE
true/false is missing from "no-reserved-keys"

### ISSUE THREAD

**Issue by szarouski** [eslint/eslint#3018]
It looks like `no-reserved-keys` doesn't catch when `true` or `false` are used as keywords, which cause ie8 to break. For following example when `no-reserved-keys` is enabled, eslint only complains about `default`.

```js
{
  true: 'text',
  false: 'password',
  default: 'hello'
 }
```

It would be better if eslint could catch when `true` or `false` are used as keys. Take a look at [similar issue](https://github.com/jscs-dev/node-jscs/pull/1233) for `jscs`. Thank you.

**Comment by nzakas** [comment]
We've run into this type of thing before (#1616). The trouble is true and false are not keywords.

IE8 has a larger set of invalid property names that we discussed adding an option for in #1616. I'm inclined to mark this as a duplicate.

**Comment by szarouski** [comment]
I'm fine with that. Thank you.

**Comment by gyandeeps** [comment]
Asp er above discussion its a duplicate.
